cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -255,8 +268,11 @@ fromShelleyTxIn (Shelley.TxIn txid txix) =
 -- Transaction outputs
 --
 
-data TxOut era = TxOut (AddressInEra era) (TxOutValue era)
+data TxOut era
+  = TxOut (AddressInEra era) (TxOutValue era)
+  deriving Generic
 
+deriving instance IsCardanoEra era => ToJSON (TxOut era)
```

Comment:
Again I think we should avoid Generic and provide a simple explicit ToJSON instance with named fields, e.g. "address" and "value".

Reply:
This was also removed in subsequent commits